Makefile

```diff
@@ -62,23 +61,34 @@ check:
 	@echo compiling for $(OSTAG) and Vertica version $(VERTICA_VERSION)
 
 .PHONY: container
-container: check
+container: check Makefile
 	@if ! $(DOCKER) image inspect $(LOCAL_IMAGE) >/dev/null 2>&1; then \
-	  $(DOCKER) container rm $(LOCAL_CONTAINER) 2>/dev/null ; \
+	  trap "$(DOCKER) container rm dblink_build 2>/dev/null || true" EXIT;  \
+	  $(DOCKER) container rm dblink_build 2>/dev/null ; \
 	  if [[ $(OSTAG) == "centos" ]]; then \
-	    $(DOCKER) run --name $(LOCAL_CONTAINER) -u 0 $(VERTICA_SDK_IMAGE) bash -c "yum install -y unixODBC-devel;" || exit 1; \
+	    $(DOCKER) run --name dblink_build -u 0 $(VERTICA_SDK_IMAGE) bash -c \
+	      " yum install -y unixODBC-devel postgresql-odbc mysql-connector-odbc" || exit 1; \
 	  else \
-	    $(DOCKER) run --name $(LOCAL_CONTAINER) -u 0 $(VERTICA_SDK_IMAGE) bash -c "apt-get update && apt-get install -y unixodbc-dev" || exit 1; \
+	    $(DOCKER) run --name dblink_build -u 0 $(VERTICA_SDK_IMAGE) bash -c \
+	      "  apt-get update \
+	      && apt-get install -y unixodbc-dev odbc-postgresql && \
+	      ln -snf /usr/bin/sudo /bin/sudo" || exit 1; \
 	  fi ; \
-	  $(DOCKER) container commit --change "USER dbadmin" $(LOCAL_CONTAINER) $(LOCAL_IMAGE); \
+	  $(DOCKER) container commit --change "USER dbadmin" dblink_build $(LOCAL_IMAGE); \
```

Comment:
why not adding a global variable for dblink_build?

Reply:
because it's a temporary label for only that target in the makefile.  Then it's deleted immediately.